public/app/features/alerting/routes.tsx

```diff
@@ -95,84 +96,118 @@ const unifiedRoutes: RouteDescriptor[] = [
   },
   {
     path: '/alerting/routes',
-    roles: () => ['Admin', 'Editor'],
+    roles: () =>
+      contextSrv.evaluatePermission(config.unifiedAlertingEnabled ? () => ['Editor', 'Admin'] : () => [], [
```

Comment:
I'm not entirely sure why this one is not using the `evaluateAccess()` function 🤔